.travis.yml

```diff
@@ -2,6 +2,8 @@ sudo: required
 language: ruby
 rvm:
   - 2.4.2
+env:
+  - SKIP_PREFLIGHT_CHECK=true
```

Comment:
Is this due to the babel issue of having babel in both `node_modules` directories? If so, would it be possible to clean up the root `node_modules` if they aren't needed for runtime? I think this might allow for some pretty sneaky build issues to arise.

Reply:
I'm not 100% sure why this is needed, but that feels right. We can't delete the root-level one because Rails needs it.